### ISSUE TITLE
Questions about commands for Imagenet and other dataset for Simple and Fast Distillation of Diffusion Models

### ISSUE THREAD

**Issue by huaishengzhu** [zju-pi/diff-sampler#20]
Hi,

Thanks for sharing your interesting work (Simple and Fast Distillation of Diffusion Models)! I have found that you provided the command to reproduce the results of Cifar10, would you mind to share the command (specific hyperparameter, learning rate and so on) for other dataset, e.g. Imagenet, afhq?

Best,

**Comment by zhyzhouu** [comment]
Thanks for your attention! The training hyperparameters for other datasets are listed in Table 6 in our paper. Specifically, to train SFD on ImageNet64, you can use
```
torchrun --standalone --nproc_per_node=4 --master_port=12345 train.py \
--dataset_name="imagenet64" --total_kimg=200 --batch=128 --lr=1e-5 \
--num_steps=4 --M=3 --afs=True --sampler_tea="dpmpp" --max_order=3 --predict_x0=True --lower_order_final=True \
--schedule_type="polynomial" --schedule_rho=7 --use_step_condition=False --is_second_stage=False
```
To train SFD-v on ImageNet64, you can use
```
torchrun --standalone --nproc_per_node=4 --master_port=12345 train.py \
--dataset_name="imagenet64" --total_kimg=800 --batch=128 --lr=1e-5 \
--num_steps=4 --M=3 --afs=True --sampler_tea="dpmpp" --max_order=3 --predict_x0=True --lower_order_final=True \
--schedule_type="polynomial" --schedule_rho=7 --use_step_condition=True --is_second_stage=False
```
For datasets like FFHQ and AFHQv2, the training hyperparameters are the same as that used for CIFAR10.
